digitalocean/app/app_spec.go

```diff
@@ -552,6 +556,14 @@ func appSpecComponentBase(componentType appSpecComponentType) map[string]*schema
 				Schema: appSpecGitLabSourceSchema(),
 			},
 		},
+		"bitbucket": {
+			Type:     schema.TypeList,
+			Optional: true,
+			MaxItems: 1,
+			Elem: &schema.Resource{
+				Schema: appSpeBitBucketSourceSchema(),
```

Comment:
Looks great. Just a tiny typo
```suggestion
				Schema: appSpecBitBucketSourceSchema(),
```

Reply:
:facepalm: Good catch!